test/powershell/Modules/Microsoft.PowerShell.Utility/Format-Table.Tests.ps1

```diff
@@ -760,4 +760,44 @@ abc abc
             $output = $obj | Format-Table | Out-String
             $output.Replace("`r","").Replace(" ",".").Replace("`n","^") | Should -BeExactly $expectedTable.Replace("`r","").Replace(" ",".").Replace("`n","^")
         }
+
+        It "Should render rows correctly when wrapped: <variation>" -TestCases @(
+            @{ variation = "right"; obj = [pscustomobject] @{A=1;B=2;Name="This`nIs some random`nmultiline content"}; expectedTable = @"
+
+A B Name
+- - ----
+1 2 This
+    Is some random
+    multiline content
+
```

Comment:
3 empty lines (here and in 2 other cases below) - are they necessary ?

Reply:
It annoys me, too, but that is currently what PowerShell outputs.  We should probably look into that as a separate issue.